java/ql/src/experimental/Security/CWE/CWE-611/XXELocal.ql

```diff
@@ -0,0 +1,29 @@
+/**
+ * @name Resolving XML external entity in local source
```

Comment:
Is there any reason to add this version? Did you need it to catch some CVE? If not, drop the local version.

Reply:
Yes i am using it to catch some cve.